frontend/src/components/EmailConfirmation.jsx

```diff
@@ -1,19 +1,24 @@
 import React, {useEffect, useState} from 'react';
 import axios from 'axios';
-import { Typography } from '@material-ui/core';
+import Login from './LoginDialog';
+import '../stylesheets/email_confirm.css';
 
 export default function EmailConfirmation(props) {
     const [message, setMessage] = useState('Confirming Email...');
+    const [email, setEmail] = useState("");
     let attempted = false;
+    const emailConfirmed = 'Email confirmed! You have applied to this event successfully!';
+    const attemptedMsg = 'Maybe the link was old? Try logging in to request a new confirmation token.'
```

Comment:
This is a Nitpick but I feel like this error message could be better (especially use of the word token could be confusing for anyone non-technical...although this might be unlikely in a hackathon)?

Reply:
Resolved! Changed message to "Maybe the link is old? Try logging in."